lib/fog/aws/credential_fetcher.rb

```diff
@@ -23,7 +21,7 @@ def fetch_credentials(options)
           if options[:use_iam_profile]
             begin
               role_data = nil
-              region = options[:region]
+              region = options[:region] || ENV["AWS_DEFAULT_REGION"]
```

Comment:
I think `AWS_REGION` might have been used before, but I think `AWS_DEFAULT_REGION` is the new environment variable: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html.